guide/14-deep-learning/labeling-text-using-doccano.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
<p>ReviewNB is showing the folder name as <strong><em>misc/tools/doccano</em>deployment_</strong>&nbsp;- not sure if it's correct or due to marup being applied to the folder. Use <code>backticks</code> to  avoid this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>